cohorts/load.py

```diff
@@ -299,7 +317,7 @@ def plot_survival(self, on, col=None, col_equals=None, how="os", threshold=None)
         results = plot_kmf(
             df=df,
             condition_col=plot_col,
-            censor_col=self.dead_col if how == "os" else self.progressed_col,
+            censor_col=self.dead_col if how == "os" else self.progressed_or_dead_col,
```

Comment:
Thinking about this a little more, this seems specific to a current use case. I think having the warning is great, but if there are two separate censor cols defined, it might be worth using the other sometimes?